src/Processors/Executors/ExecutionThreadContext.cpp

```diff
@@ -75,7 +75,7 @@ bool ExecutionThreadContext::executeTask()
 
     if (trace_processors)
     {
-        span = std::make_unique<OpenTelemetry::SpanHolder>("ExecutionThreadContext::executeTask() " + node->processor->getName());
+        span = std::make_unique<OpenTelemetry::SpanHolder>(node->processor->getName());
```

Comment:
Maybe leave some small prefix like "P:" for processors?

Reply:
The names in span in most cases are function names. So 'P:' seems a little bit wired for users to understand. I think using the processor name itself is clearly. 

If we really want to give a clear information that this span is processor, how about using the function name style by adding prefix 'Processor::' ?